zilliqa/src/consensus.rs

```diff
@@ -878,6 +879,26 @@ impl Consensus {
         ret
     }
 
+    /// Clear up anything in memory that is no longer required. This is to avoid memory leaks.
```

Comment:
I wouldn't really call it a "memory leak" - we were just explicitly and intentionally storing things that we didn't need any longer. (In my mind, a leak is when memory is kept allocated accidentally or unintentionally.)

Correspondingly I'd also name it something like `cleanup_votes` rather than memory.

Reply:
Ah yes, it is not technically a memory leak :) - okay, I will rename it!